### PR TITLE
MAUI-835176 Stacking area UG

### PR DESCRIPTION
**Pull request by snehakPV**
**Description**

Prepared UG for Stackingarea series and Stackingarea100 series

### CODE REVIEW

File: MAUI/Cartesian-Charts/StackedArea100.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+title: StackedArea100 Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here all about StackedArea100 chart support in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# StackedArea100 Chart in .NET MAUI Chart
+
+The stacked area 100% chart enables users to visually represent data points as stacked areas. In this chart, the areas are stacked on top of each other to indicate the cumulative value of the data points.
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/StackedArea100.md:12]: @snehakPV 
Both stacked area and 100 stacked area description are similar, modify like for stacked area 100 - cumulative value of the data points to 100%.

File: maui-toc.html [hunk 2]
```diff
@@ -275,6 +275,8 @@
                     <li><a href="/maui/cartesian-charts/errorbar-chart">Error Bar Chart</a></li>
                     <li><a href="/maui/cartesian-charts/OHLC">OHLC Chart</a></li>
                     <li><a href="/maui/cartesian-charts/candle">Candle Chart</a></li>
```
Comment (Saravanan-Madhesh) [inline maui-toc.html:277]: Rearrange the order.

File: MAUI/Cartesian-Charts/StackedArea.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+CategoryAxis primaryAxis = new CategoryAxis();
+chart.XAxes.Add(primaryAxis);
+NumericalAxis secondaryAxis = new NumericalAxis();
+chart.YAxes.Add(secondaryAxis);
+
+StackingAreaSeries series1 = new  StackingAreaSeries()
+{
+    XBindingPath = "Year",
+    YBindingPath = "Value",
+    ItemsSource = new ViewModel().Data1
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/StackedArea.md:69]: Simplify the code.

**Review by Saravanan-Madhesh — APPROVED** [review]
@snehakPV address review comments and raise new PR

File: maui-toc.html (hunk 2 above)
Comment (snehakPV) [reply maui-toc.html:277]: Arranged the order as mentioned.

**Pull request merged**